### PR TITLE
Frontend review location list

### PR DESCRIPTION
**Pull request by th-snu**
location review list's view is complete, actual review isn't loaded yet.
merge conflict fixed due to layout change
googlemap , searchbox is improved a lot

### CODE REVIEW

**Comment by jyeok** [comment]
Sonarcloud is failing; check and fix pls

**Comment by th-snu** [comment]
main issue is duplicated code that were copied from feed. will we make another class to separate those duplicated code?

**Comment by th-snu** [comment]
`
    reviewsToRender = reviewsToRender.map((review) => (
      <ReviewPreview
        key={`${review.id}`}
        id={review.id}
        author={review.author}
        restaurant={review.restaurant}
        menu={review.menu}
        content={review.content}
        image={review.image}
        rating={review.rating}
        date={review.date}
        tag={review.tag}
        isMine={review.isMine}
      />
    ));`

**Comment by th-snu** [comment]
if we were to remove duplicate lines, making feed into 2 components, one showing the list and one loading the list and passing it to other component would work ig

**Comment by jyeok** [comment]
That problem was resolved by yesterday, please **git pull** at `dev` and merge it into your current branch

**Comment by th-snu** [comment]
that part of code I said isn't changed at all, what you mean by it is fixed?

File: frontend/foodbook/src/store/reducers/review/reducer_review.js [hunk 1]
```diff
@@ -34,26 +34,30 @@ const reducer = (state = initialState, action) => {
 
     case actionTypes.POST_REVIEW:
       newReview = {
-        id: action.id,
```
Comment (ByungHeeCha) [inline frontend/foodbook/src/store/reducers/review/reducer_review.js]: I think we have to add both id and placeid

**Pull request merged**